jobs/validate/nvidia-spec

```diff
@@ -14,18 +14,23 @@ set -x
 ###############################################################################
 # FUNCTION OVERRIDES
 ###############################################################################
+
+# Reference: https://ubuntu.com/kubernetes/docs/gpu-workers
 function juju::deploy
 {
     tee overlay.yaml <<EOF > /dev/null
-series: $SERIES
+SERIES: $SERIES
```

Comment:
hmm?  i don't get this one?  the bundle spec says this should be lowercase
https://juju.is/docs/sdk/bundle.yaml#heading--series

Reply:
my mistake